feat(genapi): update rate limit about batches (#6275)
scaleway/docs-content: pages/generative-apis/faq.mdx

  - Audio transcription models' time to first token remains mostly constant, as they only need to process small numbers of input tokens (`30` seconds audio chunk) to generate a first output.
- Input and output size: In rough terms, total processing time is proportional to input and output size. However, for larger queries (usually above `10,000` tokens), processing speed may degrade with query size. For optimal performance, we recommend splitting queries into the smallest meaningful parts (`10` queries with `1,000` input tokens and `100` output tokens will be processed faster than `1` query with `10,000` input tokens and `1,000` output tokens).

### How long does a batch take to be processed (using Batches API)? How can I optimize this time?
We aim to process any batch in at most `24` hours. Past this delay, batch processing will be stopped and any queries remaining to be processed will not be billed.
Batches are processed in their order of creation. You can optimize time before receiving a batch output by splitting one batch into multiple smaller ones.
For example:
- Assuming a batch of `10 000` requests will take `10` hours to be processed.

> - Splitting this batch in `10` batches of `1 000` requests each will take the same time, eg. `10` hours to process all batches. However, the first batch output will be provided after `1` hour, the second one after `2` hours, etc.

Reviewer: ```suggestion
- Splitting this batch into `10 batches` of `1.000 requests` each will take the same time (e.g., `10 hours`) to process all batches. However, the first batch output will be provided after `1 hour`, the second one after `2 hours`, and so on.
```
